build.sbt

```diff
@@ -51,6 +52,9 @@ ThisBuild / githubWorkflowBuild     ~= { steps =>
 ThisBuild / githubWorkflowJavaVersions := Seq(JavaSpec.temurin("11"))
 ThisBuild / tlBspCrossProjectPlatforms := Set(JVMPlatform)
 
+ThisBuild / tlSitePublishBranch := Some("main")
+ThisBuild / tlSiteIsTypelevelProject.withRank(KeyRanks.Invisible) := Some(TypelevelProject.Affiliate)
```

Comment:
Actually it's not an affiliate project, it's an org project: it is literally under the `typelevel` org on GitHub :)
```suggestion
```